.travis.yml

```diff
@@ -4,13 +4,8 @@ jobs:
       script:
         - docker build --target all_tests -t test .
         - docker run --name todo-app-test --env TRELLO_URL --env TRELLO_KEY --env TRELLO_BOARD_ID --env TRELLO_TOKEN test
-    - stage: deploy
-      script:
-        - DOCKER_REPO="$DOCKER_USERNAME/todo-app"
-        - docker build --target prod -t "$DOCKER_REPO:$TRAVIS_COMMIT" -t "$DOCKER_REPO:latest" .
-        - docker login -u $DOCKER_USERNAME -p $DOCKER_PASSWORD
-        - docker push $DOCKER_REPO
-        - docker tag $DOCKER_REPO registry.heroku.com/tidi-ipp/web
-        - heroku container:login
-        - docker push registry.heroku.com/tidi-ipp/web
-        - heroku container:release web -a tidi-ipp
+deploy:
+  provider: script
+  script: bash ci/deploy.sh
+  on:
+    branch: main
```

Comment:
You have main here, but it looks like this repo actually has `master` as it's primary branch